### PR TITLE
Update VisualStudio.gitignore

### PR DESCRIPTION
**Pull request by denis-peshkov**
added additional files to be ignored

### CODE REVIEW

**Comment by drothmaler** [comment]
Ignoring StyleCop reports might be a good Ideas, but for `Thumbs.db` there already is `Global/Windows.gitignore`

File: VisualStudio.gitignore [hunk 1]
```diff
@@ -51,6 +51,13 @@ project.fragment.lock.json
 artifacts/
 **/Properties/launchSettings.json
 
+# StyleCop
+StyleCopReport.xml
+
+# Thumbnails created by windows
+Thumbs.db
```
Comment (shiftkey) [inline VisualStudio.gitignore]: This is covered by `Global/Windows.gitignore` - please remove it.

File: VisualStudio.gitignore [hunk 2]
```diff
@@ -51,6 +51,13 @@ project.fragment.lock.json
 artifacts/
 **/Properties/launchSettings.json
 
+# StyleCop
+StyleCopReport.xml
```
Comment (shiftkey) [inline VisualStudio.gitignore]: Any documentations or discussions to explain the context behind this change?

File: VisualStudio.gitignore (hunk 1 above)
Comment (denis-peshkov) [reply VisualStudio.gitignore]: Removed

File: VisualStudio.gitignore (hunk 2 above)
Comment (denis-peshkov) [reply VisualStudio.gitignore]: Unfortunately I can't found out any documents described the output files...
But this file appears very often in our projects, so I propose add it for now and add/change in future, when will have more info.

**Pull request merged**